docs/howto/getting-started/accessing-cc-rest-api.md

```diff
@@ -0,0 +1,138 @@
+# Accessing the Cleura Cloud REST API
+
+{{brand}} provides a REST API, which you can take advantage of with a 
+tool like `curl`. But before you do, you must create a token for the 
+current session. For that, you only need to have an
+[account in {{brand}}](/howto/getting-started/create-account).
+
+## Creating a token
+
+To create a token, have your username (`your_cc_username`) and password 
+(`your_cc_password`) to the {{brand}} handy, and type:
+
+```bash
+curl -H "Content-Type: application/json" -X POST -d \
+    '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
+    https://rest.cleura.cloud/auth/v1/tokens
+```
+
+Provided you typed your username and password correctly and that there 
+were no connection issues to the remote endpoint, you will get a JSON 
+object with a string (`token`) that holds your session token 
+(`a_token_for_you`):
```

Comment:
Tokens are 32 character mixed-case random strings; you can generate a non-functional example token with `pwgen -c 32 1`.